src/controllers/hid/hidcontroller.cpp

```diff
@@ -293,8 +293,32 @@ void HidController::sendBytesReport(QByteArray data, unsigned int reportID) {
         }
     } else {
         controllerDebug(result << "bytes sent to" << getName()
-                 << "serial #" << hid_serial
-                 << "(including report ID of" << reportID << ")");
+                               << "serial #" << hid_serial
+                               << "(including report ID of" << reportID << ")");
+    }
+}
+
+void HidController::sendFeatureReport(
+        const QList<int>& dataList, unsigned int reportID) {
+    QByteArray dataArray;
+    for (const int datum : dataList) {
+        dataArray.append(datum);
+    }
+
+    // Append the Report ID to the beginning of dataArray[] per the API..
+    dataArray.prepend(reportID);
```

Comment:
Prepending doesn't make sense here, it requires reshuffling the contents of QByteArray. Append everything in order.

Reply:
Done